/**:
  ros__parameters:
    # Filtering parameters
    enable_object_filtering: true
    enable_pointcloud_filtering: true
    max_filter_distance: 5.0  # [m]
    pointcloud_safety_distance: 1.0  # [m]
    object_classification_radius: 50.0  # [m]

    # RTC recreation on vehicle stop
    stop_velocity_threshold: 0.001  # [m/s] Velocity threshold to consider vehicle as stopped
